src/main/java/net/fabricmc/mappingio/MappingReader.java

```diff
@@ -196,20 +212,33 @@ public static void read(Path path, MappingVisitor visitor) throws IOException {
 	 * @param visitor The receiving visitor.
 	 * @throws IOException If reading fails.
 	 */
+	@Deprecated
 	public static void read(Path path, MappingFormat format, MappingVisitor visitor) throws IOException {
+		read(path, format, visitor, new ThrowingErrorCollector(Severity.ERROR));
```

Comment:
I think the existing behavior would match `Severity.INFO`, correct? Better keep the same behavior.

Reply:
Good catch. INFO indicates inconsequential issues though, so I'd rather drop it down to `WARNING` instead